lib/matplotlib/patches.py

```diff
@@ -1667,7 +1667,7 @@ def segment_circle_intersect(x0, y0, x1, y1):
         # circle in the same way that it is relative to the desired ellipse.
         box_path = Path.unit_rectangle()
         box_path_transform = (transforms.BboxTransformTo(self.axes.bbox)
-                              - self.get_transform())
+                              + self.get_transform().inverted())
```

Comment:
this is less precise (box_path gets some ~1e-19 errors) but is needed to make Arc work...